invenio_db/alembic/dbdbc1b19cf2_create_transaction_table.py

```diff
@@ -40,8 +41,9 @@ def upgrade():
         sa.Column('issued_at', sa.DateTime(), nullable=True),
         sa.Column('id', sa.BigInteger(), nullable=False),
         sa.Column('remote_addr', sa.String(length=50), nullable=True),
-        sa.PrimaryKeyConstraint('id')
     )
+    op.execute(CreateSequence(Sequence("transaction_id_seq")))
```

Comment:
This will work only for Postgres - can we investigate an option with `autoincrement`?